docs/yt_idv.rendering_contexts.rst

```diff
@@ -12,7 +12,7 @@ These are the top-level rendering contexts available in ``yt_idv``.
    :undoc-members:
    :show-inheritance:
 
-.. autoclass:: yt_idv.rendering_contexts.egl_context.OSMesaRenderingContext
+.. autoclass:: yt_idv.rendering_contexts.osmesa_context.OSMesaRenderingContext
```

Comment:
so this class is actually throwing an error while building and gets left off the docs, even with this fix. That error comes from how we're setting `os.environ["PYOPENGL_PLATFORM"] = "egl"` in our conf.py -- after doing that , the OpenGL imports in `yt_idv.rendering_contexts.osmesa_context` will error. Dont see a way to fix that -- if we switch the conf.py platform to `osmesa`, then the `egl_context` will probably error?